rasa/core/channels/twilio.py

```diff
@@ -123,7 +123,7 @@ def blueprint(self, on_new_message):
             else:
                 logger.debug("Invalid message")
 
-            return response.text("success")
+            return response
```

Comment:
did you actually try if this works? `response` is a python module, should probably be 

```suggestion
            return response.text("", status=204)
```

Reply:
its work but not a proper way, because sanic response they need to return something